Source/WPEFramework/PluginHost.cpp

```diff
@@ -316,6 +317,24 @@ POP_WARNING()
 
             ExitHandler::StartShutdown();
         }
+        else if (signo == SIGSEGV) {
+
+            // From here on we do the best we can do. Have no clue what failed, try to log as much as possible and on
+            // a subsequent segmentation fault, just handle it the old fashion way. The root cause has been logged
+            // by than!
+            sigaction(SIGSEGV, &_originalSegmentationHandler, nullptr);
```

Comment:
Just thinking wouldn't it be better to turn this and the next line around? I don't know if on Linux if we get here the process is frozen but on Windows I think that is not the case and a different thread could cause an access violation then trigger the original handler and just abort the process. Or does this work completely different on Linux?

Reply:
??? This is not stacked in any way so setting back the handlers in what ever order does not make a difference. Reinstalling the basic handler is mandatory before we proceed, otherwise we could get recursively called over and over again :-) So I do not thin there is a change required here.